app/pktgen-stats.c

```diff
@@ -533,14 +534,14 @@ pktgen_process_stats(struct rte_timer *tim __rte_unused, void *arg __rte_unused)
         rate = &info->rate_stats;
         prev = &info->prev_stats;
 
-        rate->ipackets  = curr->ipackets - prev->ipackets;
-        rate->opackets  = curr->opackets - prev->opackets;
-        rate->ibytes    = curr->ibytes - prev->ibytes;
-        rate->obytes    = curr->obytes - prev->obytes;
-        rate->ierrors   = curr->ierrors - prev->ierrors;
-        rate->oerrors   = curr->oerrors - prev->oerrors;
-        rate->imissed   = curr->imissed - prev->imissed;
-        rate->rx_nombuf = curr->rx_nombuf - prev->rx_nombuf;
+        rate->ipackets  = (curr->ipackets - prev->ipackets) * Billion / (double)elapsed_ns + 0.5;
```

Comment:
I can never remember the order of presidents, can you add () to make the order clear.